src/themes/baseTheme.ts

```diff
@@ -44,7 +44,7 @@ export const baseTheme = createTheme({
     },
     components: {
       button: {
-        borderRadius: { value: '{radii.large}' },
+        borderRadius: { value: '{radii.medium}' },
```

Comment:
Reset base button size to have a medium border radius. borderRadius isn't a type on our button size variations so I can't set it in our theme, so I set the large button size to have a large border radius in base.scss instead.